sites/docs/pages/components/button-group.md

```diff
@@ -223,6 +267,11 @@ group by all
     description="SQL where fragment to filter options by (e.g., where sales > 40000)"
     options="SQL where clause"
 />
+<PropListing 
+    name="display"
+    description="Displays tabs with button functionality"
+    options="tabs"
```

Comment:
`defaultValue="buttons"`?

I know you have not actually set a default value but it's confusing to only have one value, and it not be set

(otherwise we should have done something like `isTabs=true`, default false)

Reply:
In https://github.com/evidence-dev/evidence/pull/2483, we introduce `defaultValue="buttons"` in button groups. But I Agree, we need to add it to the docs. I will add it to the docs for 2483 since it is more relevant.

will also add the clickable option!

Thanks